### DISCUSSION TITLE
wouldn't M109 be more appropriate than m104?

### DISCUSSION THREAD

**Discussion by PaintOnBrush** [5axes/Calibration-Shapes#77] [General]
wouldn't "M109 R" be more appropriate than "m104 S" in the tempfantower.py script?

this would wait until the temp gets to the temperature before starting the layers.

**Comment by 5axes** [comment]
I don't think so as generaly we start from the higher temperature to the lower ,  M109 are not usefull because we are already at an higher value so the 104 function doesn't wait anything.

**Reply by PaintOnBrush** [reply]
I find it useful to measure the ooze factor between the cooling dwell time from higher temperature to lower (I might have accentuated the top ooze blob a bit)
![image](https://user-images.githubusercontent.com/2153952/149318223-768ceb33-44be-4767-b102-d69bb59ef419.png)
which would result in
![image](https://user-images.githubusercontent.com/2153952/149322584-36b36e71-dd4e-444c-a964-57edb802a3a9.png)
the ooze blobs' diameter are easily measurable with a set of calipers too.

I've found some materials to not ooze at all at certain temps and still make strong prints with this method.

**Comment by 5axes** [comment]
As in Cura the temperature change inside a print ( IE from the initial layer printing temperature and the printing temperature ) is done with a **M104 S** command line , I don't see exactly the benefits to test the use of the **M109 R** instruction and to measure the ooze blob default which should be not present if you use the M104 S instruction ? 
![image](https://user-images.githubusercontent.com/11015345/149327478-34e9bf6a-40fd-4d88-9de7-05c6e057bdba.png)
